porting/common-system-build-errors.rst

```diff
@@ -14,6 +14,19 @@ All APKs and Java libraries should be removed from the Makefiles in ``vendor/[ma
 
 Also check the vendor folders of your dependencies.
 
+Undefined or missing bluetooth headers
+-------------------------------------
+
+.. code-block:: guess
+
+    *TODO: example*
+
+Some devices requires bluetooth when compiling, which aren't synced for default by Halium. This can be fixed adding LineageOS's repo to the device manifest:
+
+.. code-block:: guess
```

Comment:
Instead of using the code-block directive directly, place two colons at the end of your last paragraph and indent the code block. For example:

```
... to the device manifest::

    <project path="system/bt" name="android_system_bt" remote="los" revision="cm-14.1" />
```